Tools/WinMLRunner/src/Run.cpp

```diff
@@ -561,6 +563,7 @@ int run(CommandLineArgs& args,
             ConcurrentLoadModel(modelPaths, args.NumThreads(), args.ThreadInterval(), true);
             return 0;
         }
+        traceHelper.Start();
```

Comment:
>traceHelper [](http://example.com/codeflow?start=8&length=11)

This is only enabled for non-concurrent model load. Can we add a message to indicate that it is not going to actualy trace in this configuration?